### PR TITLE
Dsg 934 agency portal as an agent i want to be able to sort notes

### PR DESCRIPTION
**Pull request by g8shen**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

dropdowns to sort notes and removing the filter button.

## Screenshot of the affected changes

New sorting in agency:
https://github.com/Nuvalence/dsgov-web/assets/39806680/b1f00789-0c75-4de2-b2d5-7e4b0bc4f7e7

Updated component in public/employer still looks the same:
Screenshot 2023-11-17 at 11 02 42 AM
Screenshot 2023-11-17 at 11 03 19 AM

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/data-access/http/src/lib/models/pagination/paging-request.model.ts [hunk 1]
```diff
@@ -2,8 +2,18 @@
 import { ActivatedRoute, Params, Router } from '@angular/router';
 import { SchemaModel } from '../base.model';
 
+export enum SortOrderButton {
```
Comment (davidyoung77) [inline libs/shared/data-access/http/src/lib/models/pagination/paging-request.model.ts:5]: This looks like a display specific type and would be better stored in a model with the nuverial component, please correct me if I am wrong

**Review by davidyoung77 — COMMENTED** [review]
Thanks, I have a couple comments

File: libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts [hunk 2]
```diff
@@ -36,13 +36,17 @@ export class NuverialSelectorButtonDropdownComponent {
    * Input of INuverialSelectOption objects, the displayTextValue key will be displayed in a item inside the menu
    */
   @Input() public menuItems!: INuverialSelectOption[];
+  /**
+   * String that specifies the icon
+   */
+  @Input() public iconName!: string;
```
Comment (davidyoung77) [inline libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts:42]: Make the dropdown the default here then you wont need the changes in public or employer, this should also be typesafe

File: libs/shared/data-access/http/src/lib/models/pagination/paging-request.model.ts (hunk 1 above)
Comment (g8shen) [reply libs/shared/data-access/http/src/lib/models/pagination/paging-request.model.ts:5]: This is being used for the the toggle order button, not in the nuverial dropdown component if that's what you where referring too.

File: libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts (hunk 2 above)
Comment (davidyoung77) [reply libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts:42]: Got it, we should give this a default though

File: libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts [hunk 3]
```diff
@@ -36,13 +36,17 @@ export class NuverialSelectorButtonDropdownComponent {
    * Input of INuverialSelectOption objects, the displayTextValue key will be displayed in a item inside the menu
    */
   @Input() public menuItems!: INuverialSelectOption[];
+  /**
+   * String that specifies the icon by default it is arrow_drop_down
+   */
+  @Input() public iconName = 'arrow_drop_down';
```
Comment (davidyoung77) [inline libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.ts:42]: Lets make this typesafe please
```suggestion
  @Input() public iconName: 'arrow_drop_down' = 'arrow_drop_down';
```

**Review by davidyoung77 — APPROVED** [review]
Looks good to me, pending comment

**Review by davidyoung77 — APPROVED** [review]

**Pull request merged**